docs/features/bank.md

```diff
@@ -7,30 +7,36 @@ This module is mainly used to transfer coins between accounts、query account ba
 
 1. Query the coin_type configuration of a certain token:
     ```bash
-    iriscli bank coin-type [coin-name]
+    iriscli bank coin-type <coin-name>
     ```
+    
+    example:
+    ```bash
+    iriscli bank coin-type iris
+    ```
+    
     Refer to [Query of coin_type](./basic-concepts/coin-type.md#query-of-coin_type) for details.
 
 2. Query account
 
-    Query the account information of a certain account address, including the balance, the public key, the account number and the transaction number.
+    Query the account information of a certain account address, including the balance, the public key, the account number and the transaction sequence.
     ```bash
-    iriscli bank account [account address]
+    iriscli bank account <account_address>
     ```
 
 3. Transfer between accounts
 
     For example, transfer 10iris from account A to account B:
     ```bash
-    iriscli bank send --to [address of wallet B] --amount=10iris --fee=0.3iris --from=[key name of wallet A] --chain-id=[chain-id]
+    iriscli bank send --to=<address_of_wallet_B> --amount=10iris --fee=0.3iris --from=<key_name_of_wallet_A> --chain-id=<chain-id>
     ```
     IRISnet supports multiple tokens in circulation, and in the future IRISnet will be able to include multiple tokens in one transaction -- tokens can be any coin_type registered in IRISnet. 
 
 4. Sign transactions generated offline
 
-    To improve account security, IRISnet supports offline signing of transactions to protect the account's private key. In any transaction, you can build an unsigned transaction using the flag --generate-only=true. Use transfer transactions as an example:
+    To improve account security, IRISnet supports offline signing of transactions to protect the account's private key. In any transaction, you can build an unsigned transaction using the flag --generate-only. Use transfer transaction as an example:
```

Comment:
`Use transfer transaction as an example:` -> `Take transfer transaction for example:`

Reply:
fixed